src/controllers/registerGuildController.ts

```diff
@@ -0,0 +1,26 @@
+import { Guild } from 'discord.js'
+import { PrismaGuildRepository } from '@/repositories/prisma/prismaGuildRepository'
+import { RegisterNewGuildUseCase } from '@/useCases/registerNewGuildUseCase'
+import { PrismaHomebrewRepository } from '@/repositories/prisma/prismaHomebrewRepository'
+
+export const registerGuildController = async (guild: Guild): Promise<void> => {
```

Comment:
We have to discuss whether we are going to keep OOP or functional programming. And if we keep both, in which case we use each of them.